sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlAsyncClient.java

```diff
@@ -120,9 +133,12 @@ HttpPipeline getHttpPipeline() {
      * {@link KeyVaultRoleScope role scope} and above.
      *
      * @param roleScope The {@link KeyVaultRoleScope role scope} of the {@link KeyVaultRoleDefinition role definitions}.
+     *
      * @return A {@link PagedFlux} containing the {@link KeyVaultRoleDefinition role definitions} for the given
      * {@link KeyVaultRoleScope role scope}.
-     * @throws NullPointerException if the {@link KeyVaultRoleScope role scope} is {@code null}.
+     *
+     * @throws KeyVaultAdministrationException If the given {@code roleScope} is invalid.
```

Comment:
If this is just a failed request exception, a separate type should not be necessary. Azure errors are standard. At least in .NET, we all use Azure.Core.RequestFailedException and rarely derive or create our own exceptions unless there's information to add.